webpack.dev.config.js

```diff
@@ -27,8 +27,12 @@ var cf = {
 			exclude: /node_modules/,
 			loaders: ["raw-loader"]
 		},{
+			test: /\/page1.*\.html$/,
```

Comment:
Thank you, just  to be clear, Do have I to remove completely ngTemplate using only html-loader ? Do I have to do same as you with page1 but for each other component ?